### DISCUSSION TITLE
Renaming this repository to `rust-design-patterns` or alike?

### DISCUSSION THREAD

**Discussion by simonsan** [rust-unofficial/patterns#113] [Ideas]
Not sure how people think about it, but I would propose renaming the repository to `rust-patterns` to make it more easily distinguishable in forks but also for the book domain, which is now: https://rust-unofficial.github.io/patterns/

Other naming ideas:
- `rust-design-patterns`
- `design-patterns`

Thoughts?

**Comment by marcoieni** [comment]
Since `rust` is already present in the organization name, I would not repeat it in the repository name, but that's just personal taste.
I am open to rename this repo to `design-patterns`

**Reply by simonsan** [reply]
Agreed, makes sense to spare the doubled Rust in it.

**Reply by ehiggs** [reply]
From the peanut gallery: I think rust/compiler wouldn't be appropriate and if I fork the book I will have "design-patterns" as a local repo which doesn't say much.

**Reply by simonsan** [reply]
> 
> 
> From the peanut gallery: I think rust/compiler wouldn't be appropriate and if I fork the book I will have "design-patterns" as a local repo which doesn't say much.

Agreed as well, what is your proposal for a good name then?

**Reply by ehiggs** [reply]
Your `rust-design-patterns` is a good name imo

**Reply by marcoieni** [reply]
> From the peanut gallery: I think rust/compiler wouldn't be appropriate and if I fork the book I will have "design-patterns" as a local repo which doesn't say much.

https://github.com/rust-lang/book :stuck_out_tongue:

**Reply by marcoieni** [reply]
Anyway for me no problem in renaming this `rust-design-patterns`

**Reply by simonsan** [reply]
Someone knows if there would be a redirect for the github.io or is the redirect only for the repository then?
